### ISSUE TITLE
MultiLineArrayComma sniff doesn't work with comments at the end

### ISSUE THREAD

**Issue by rweich** [escapestudios/Symfony2-coding-standard#24]
adding a comment after the last comma makes the sniff fail again
```php
$array = [
    'foo' => 1, //bar
];
```

**Comment by Soullivaneuh** [comment]
:+1:

**Comment by Soullivaneuh** [comment]
This work:

```php
->add('environment', null, [], 'choice', [
    'choices'           => $this->environmentsChoices,
    'choices_as_values' => true, // TODO: Remove it on SF 3.0, default value.
])
```

This doesn't:

```php
->add('environment', null, [], 'choice', [
    'choices'           => $this->environmentsChoices,
    'choices_as_values' => true,
])
```

**Comment by djoos** [comment]
Fixed in #25!

Kind regards,
David
